test/evp_libctx_test.c

```diff
@@ -439,6 +475,8 @@ static int test_cipher_reinit_partialupdate(int test_id)
         goto err;
 
     in_len = EVP_CIPHER_get_block_size(cipher) / 2;
+    if (!TEST_int_ge(in_len, 0))
```

Comment:
int_gt()

Reply:
fixed

Reply:
Ha, this causes failures for RC4 - there needs to be check for EVP_CIPHER_get_ivlen() == 0 -> skipping the 3rd testcase.